config/initializers/z_plugin_parsers.rb

```diff
@@ -1,25 +1,19 @@
-class ParserRegistrator
-  def self.register_fact_parser(type, parser)
-    if (old_parser = FactParser.parser_for(type)) && old_parser != FactParser.parsers.default
-      Rails.logger.warn("WARNING: Parser #{old_parser} for type #{type} is replaced with #{parser}")
-    end
-
-    FactParser.register_fact_parser(type, parser)
-  end
-end
+# Puppet, the default parser
```

Comment:
Unrelated and fine in another PR but I'd consider renaming this file to `fact_parsers.rb`.

Reply:
I don't think it's a good idea. The initializer will be called before Foreman initializer and that going to fail into `NoMethodError` or similar.

Reply:
It could also be `z_fact_parsers` but I think `plugin_parsers` isn't entirely covering the content. Again, not needed for this PR but something for future refactors.